actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -90,18 +100,18 @@ class ActionCable.Connection
     open: ->
       ActionCable.log("WebSocket onopen event")
       @disconnected = false
```

Comment:
Guess that ^ doesn't remove the need to check `disconnected` in the message handler, but I find it clearer since it consolidates the logging and removes the early `return`.